tests/app_client/automation.py

```diff
@@ -0,0 +1,182 @@
+from typing import Tuple
+from abc import ABCMeta, abstractmethod
+from urllib.parse import urljoin
+from enum import IntEnum
+from functools import reduce
+import operator
+
+from requests import Session
+
+
+class ConditionFlag(IntEnum):
+    RUN_ONCE = 1
+    RUN_FOR = 2
+    NOT_RUN = 3
+
+
+class Button(IntEnum):
+    LEFT = 1
+    RIGHT = 2
+
+
+class Rules:
+    go_right = [
+        ["button", Button.RIGHT.value, True],
```

Comment:
What's this last parameter? I think a small docstring would be helpful here

Reply:
These rules are defined on https://speculos.ledger.com/user/automation.html I have described this on the docstring